MeshLib/Elements/Element.cpp

```diff
@@ -239,4 +244,106 @@ bool isPointInElementXY(MathLib::Point3d const& p, Element const& e)
          MeshLib::MeshElemType2String(e.getGeomType()).c_str());
     return false;
 }
+
+std::vector<std::size_t> findElementsInRadius(Element const& start_element,
```

Comment:
I'd opt for a specific file for this algorithm. Once we have 100 of them we can group them together.